viam-cartographer/src/carto_facade/carto_facade.cc

```diff
@@ -628,10 +637,9 @@ int CartoFacade::Stop() {
 };
 
 void CartoFacade::AddSensorReading(const viam_carto_sensor_reading *sr) {
-    std::string sensor = to_std_string(sr->sensor);
-    if (config.sensors[0] != sensor) {
-        VLOG(1) << "expected sensor: " << sensor << " to be "
-                << config.sensors[0];
+    if (biseq(config.component_reference, sr->sensor) == false) {
```

Comment:
What does this do?

Reply:
this tests that two bstring s are not equal